### PR TITLE
chore: fix crystal-nightly compatibility

### PR DESCRIPTION
**Pull request by veelenga**
* this specific test fails in master, [example](https://github.com/crystal-ameba/ameba/actions/runs/4215042567/jobs/7315902519)

### CODE REVIEW

**Comment by veelenga** [comment]
Hard to say how to fix the formatter issue here. Crystal now reformats the docs and bad examples could be reformatted to the good ones.

In our case it tries to reformat this

https://github.com/crystal-ameba/ameba/blob/031c1daf58a4c39a5f1863da8c0875141a604546/src/ameba/rule/lint/missing_block_argument.cr#L7-L9

into a valid case. See https://github.com/crystal-lang/crystal/pull/12951

**Comment by Sija** [comment]
> Hard to say how to fix the formatter issue here. Crystal now reformats the docs and bad examples could be reformatted to the good ones.
> 
> In our case it tries to reformat this
> 
> https://github.com/crystal-ameba/ameba/blob/031c1daf58a4c39a5f1863da8c0875141a604546/src/ameba/rule/lint/missing_block_argument.cr#L7-L9
> 
> into a valid case. See [crystal-lang/crystal#12951](https://github.com/crystal-lang/crystal/pull/12951)

These will need to use 4-spaces indentation instead of the backticks, since the indented code is not touched by the Crystal formatter.

File: src/ameba/severity.cr [hunk 1]
```diff
@@ -41,7 +41,7 @@ module Ameba
     def self.parse(name : String)
       super name
     rescue ArgumentError
-      raise "Incorrect severity name #{name}. Try one of #{values}"
+      raise "Incorrect severity name #{name}. Try one of available names"
```
Comment (Sija) [inline src/ameba/severity.cr:44]: If we're not listing the names, I'd reformat this to:

```suggestion
      raise "Incorrect severity name: #{name}"
```

File: src/ameba/severity.cr [hunk 2]
```diff
@@ -41,7 +41,7 @@ module Ameba
     def self.parse(name : String)
       super name
     rescue ArgumentError
-      raise "Incorrect severity name #{name}. Try one of #{values}"
+      raise "Incorrect severity name #{name}. Try one of #{values.map(&.to_s).join(", ")}"
```
Comment (Sija) [inline src/ameba/severity.cr:44]:
```suggestion
      raise "Incorrect severity name #{name}. Try one of: #{values.map(&.to_s).join(", ")}"
```

**Review by Sija — APPROVED** [review]

**Comment by Sija** [comment]
Might land in 1.4.2 too, btw.

**Comment by veelenga** [comment]
Cool. Will release it during the coming week.

**Pull request merged**